src/v2/Apps/Consign/Routes/SubmissionFlow/UploadPhotos/UploadPhotos.tsx

```diff
@@ -1,6 +1,12 @@
 import React, { FC } from "react"
 import { Box } from "@artsy/palette"
+import { SubmissionStepper } from "v2/Apps/Consign/Components/SubmissionStepper"
 
 export const UploadPhotos: FC = () => {
-  return <Box mt={50}>Upload Photos step</Box>
+  return (
+    <>
+      <SubmissionStepper currentStep="Upload Photos" />
+      <Box mt={50}>Upload Photos step</Box>
```

Comment:
ditto

Reply:
fixed in following PR https://github.com/artsy/force/pull/8545